vllm/attention/backends/flashinfer.py

```diff
@@ -127,6 +127,7 @@ def __post_init__(self):
             raise ValueError(
                 f"Only {supported_head_sizes} are supported for head_dim,",
                 f"received {self.head_dim}.")
+        self.is_profile_run = is_block_tables_empty(self.block_tables)
```

Comment:
@LiuXiaoxuanPKU I think this line is buggy, `self.block_tables` here is always a tensor so `is_block_tables_empty` should always return False. From what I've checked this will happen to work for flashinfer v0.1.2, but not v0.1.3, which will raise `RuntimeError: CHECK_EQ(paged_kv_indptr.size(0), batch_size + 1) failed. 1 vs 257` during profile run. (This is due to the `if self.is_profile_run` block never being run.)

I tried a quick fix of patching `is_block_tables_empty` by checking if block_tables has num_el == 0, which will pass the profile run fine. But this introduces logic issues for prefill, which obviously has a 0 length block table as well. Not sure what the best fix for this is, just wanted to raise this concern here.